de.capypara.FieldMonitor.json

```diff
@@ -0,0 +1,336 @@
+{
+    "id": "de.capypara.FieldMonitor",
```

Comment:
how can we correlate the domain with the app?

Reply:
Ah right, I forgot that my other app submissions didn't go via this domain yet. I can't verify the domain right now, as login on flathub.org seems to be broken right now, but I'll do asap.

Reply:
Right, I can login now but I can't verify a non published app. I guess here's proof the domain is mine:
https://capypara.de/flathub-submission.txt